src/playwrightTestServer.ts

```diff
@@ -313,10 +327,15 @@ export class PlaywrightTestServer {
   }
 
   private async _createTestServer(): Promise<TestServerConnection | null> {
-    const args = [this._model.config.cli, 'test-server', '-c', this._model.config.configFile];
+    const cwd = this._model.config.workspaceFolder;
+    const args = [
+      path.relative(cwd, this._model.config.cli),
```

Comment:
This one deserves the same comment as in the `_configPath()` method. I wonder whether it's better to rename `_configPath()` into `_relativePath()` and reuse? Or perhaps `_normalizePaths()` that would return `{ cwd, cli, config }` triplet with all the right relativeness/casing and have a single comment explaining the situation?

Reply:
love the triplet idea!